src/index.js

```diff
@@ -22,10 +24,41 @@ import tabsSVG from '@eeacms/volto-tabs-block//icons/tabs.svg';
 import rightSVG from '@plone/volto/icons/right-key.svg';
 import downSVG from '@plone/volto/icons/down-key.svg';
 
-export default (config) => {
+const messages = defineMessages({
+  Tabs: {
+    id: 'tabs',
+    defaultMessage: 'Tabs',
+  },
+  Default: {
+    id: 'default',
+    defaultMessage: 'Default',
+  },
+  AccordionResponsive: {
+    id: 'accordion-responsive',
+    defaultMessage: 'Accordion responsive',
+  },
+  HorizontalResponsive: {
+    id: 'horizontal-responsive',
+    defaultMessage: 'Horizontal responsive',
+  },
+  CarouselHorizontal: {
+    id: 'carousel-horizontal',
+    defaultMessage: 'Carousel horizontal',
+  },
+  CarouselVertical: {
+    id: 'carousel-vertical',
+    defaultMessage: 'Carousel vertical (prototype)',
+  },
+  DefaultTitle: {
+    id: 'default-title',
+    defaultMessage: 'Tab {tabTitle}',
+  },
+});
+
+export default (config, props) => {
   config.blocks.blocksConfig[TABS_BLOCK] = {
     id: TABS_BLOCK,
-    title: 'Tabs',
+    title: props.intl.formatMessage(messages.Tabs),
```

Comment:
@macagua This still doesn't work. Volto fails to start with:
```
/opt/sandbox/clean-volto-project/src/addons/volto-tabs-block/src/index.js:61
    title: props.intl.formatMessage(messages.Tabs),
^

TypeError: Cannot read properties of undefined (reading 'intl')

  59 |   config.blocks.blocksConfig[TABS_BLOCK] = {
  60 |     id: TABS_BLOCK,
> 61 |     title: props.intl.formatMessage(messages.Tabs),
  62 |     icon: tabsSVG,
  63 |     group: 'common',
  64 |     edit: TabsEdit,

    at module.exports../src/addons/volto-tabs-block/src/index.js.__webpack_exports__.default (/opt/sandbox/clean-volto-project/src/addons/volto-tabs-block/src/index.js:61:1)
    at /opt/sandbox/clean-volto-project/build/webpack:/tmp/tmp-102795-Ixt0cAFUJqt5-.js:17:1
    at /opt/sandbox/clean-volto-project/build/webpack:/tmp/tmp-102795-Ixt0cAFUJqt5-.js:35:1
    at Array.reduce (<anonymous>)
    at load (/opt/sandbox/clean-volto-project/build/webpack:/tmp/tmp-102795-Ixt0cAFUJqt5-.js:35:1)
    at Module../node_modules/@plone/volto/src/config/index.js (/opt/sandbox/clean-volto-project/build/webpack:/node_modules/@plone/volto/src/config/index.js:224:1)
    at __webpack_require__ (/opt/sandbox/clean-volto-project/build/webpack:/webpack/bootstrap:754:1)
    at fn (/opt/sandbox/clean-volto-project/build/webpack:/webpack/bootstrap:59:1)
    at Module../node_modules/@plone/volto/src/server.jsx (/opt/sandbox/clean-volto-project/build/server.js:114429:77)
    at __webpack_require__ (/opt/sandbox/clean-volto-project/build/webpack:/webpack/bootstrap:754:1)

sswp> !!! Script exited with code 1
```

Reply:
@macagua I don't think you can use intl in index.js applyConfig

Reply:
@avoinea let me checkout this error, thanks you for the patience!!!